org.opent2t.sample.multisensor.superpopular/com.wink.sensorpod/js/tests/invalidDeviceData.json

```diff
@@ -0,0 +1,118 @@
+{   "base_state" :
+    {   "data" :
+        {
+            "last_event": {
+                "brightness_occurred_at": null,
+                "loudness_occurred_at": null,
+                "vibration_occurred_at": null
+            },
+            "object_type": "sensor_pod",
+            "object_id": "123456",
+            "uuid": "01234567-0123-0123-0123-0123456789ab",
+            "icon_id": null,
+            "icon_code": null,
+            "desired_state": {},
+            "last_reading": {
+                "temperature": 22.77777777777778,
+                "temperature_updated_at": 1488847336.5081685,
+                "motion": true,
+                "motion_updated_at": 1488847336.5081685,
+                "battery": 0.0,
+                "battery_updated_at": 1488847336.5081685,
+                "tamper_detected": false,
+                "tamper_detected_updated_at": 1488847031.0248313,
+                "motion_true": "N/A",
+                "motion_true_updated_at": null,
+                "tamper_detected_true": "N/A",
+                "tamper_detected_true_updated_at": null,
+                "connection": true,
+                "connection_updated_at": 1488847336.5081685,
+                "agent_session_id": null,
+                "agent_session_id_updated_at": null,
+                "connection_changed_at": 1488846785.7296247,
+                "motion_changed_at": "undefined",
+                "battery_changed_at": "N/A",
+                "temperature_changed_at": 1488847336.5081685,
+                "tamper_detected_changed_at": 1488847025.6141558,
```

Comment:
Would love to trim it down to just invalid data for *_changed_at but i wasnt sure which ones were absolutely needed for the translator and which ones were extraneous here.